molecule/docker/playbook.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Run role
+  hosts: all
+  any_errors_fatal: true
+  roles:
+    - ansible-prometheus
+  vars:
+    prometheus_docker_install: true
+    prometheus_version: latest
```

Comment:
Please lock version and do not use `latest`. Using `latest` is causing flaky tests in CI due to GitHub rate limiter (`latest` needs to send a request to GitHub API)